app/views/business_categories/show.html.haml

```diff
@@ -1,15 +1,32 @@
 %header.entry-header
-  %h1.entry-title= 'Företagstyp: ' "#{@business_category.name}"
+  %h1.entry-title
+    Hundföretag av typen
+    = @business_category.name
 .post-title-divider
   %span
 .entry-content
-  %p
-    %b Namn:
-    = @business_category.name
-  %p
-    %b Beskrivning:
-    = @business_category.description
-.center
-  = link_to 'Redigera', edit_business_category_path(@business_category)
-  \|
-  = link_to 'Alla kategorier', business_categories_path
+  - unless @business_category.description.nil?
+    %p
+      %b Beskrivning:
+      = @business_category.description
+  %table
+    %thead
+      %tr
+        %th Namn
+        %th Län
+        - if current_user.try(:admin)
+          %th Org nr
+          %th
+          %th
+    %tbody
+      - @companies.each do |company|
+        - if company_complete? company
+          - company.categories.each do |category|
+            - if category.name == @business_category.name
+              %tr.company
+                %td= link_to company.name, company
+                %td= company.region
+                - if current_user.try(:admin)
+                  %td= company.company_number
+                  %td= link_to 'Edit', edit_company_path(company)
+                  %td= link_to 'Delete', company, method: :delete, data: { :confirm => 'Are you sure?' }
```

Comment:
That association needs to be removed.  The association between a company and business categories is a remnant from the 'first' descriptions of the system, when _companies_ were the entities that were applying for membership.  That changed and now it's _individuals_ [users] that apply for membership (with a membership_application).


The associations are:
-  a user has (fills out) a membership application
   -  a membership application has many business categories (these are the skills/business services that a user is certified to offer)
        - a membership application has a company; the user is employed by the company

- a company can have many membership_applications (the mirror of the above)


I poked around at this, and this is what I've come up with:
1.  In`BusinessCategory` we need to add `has_many :membership_applications` so we can use the association from that direction
2. in `BusinessCategoriesController`:
``` 
 def show
    @companies = Company.includes(membership_applications: :business_categories).where(business_categories: { id: @business_category.id }).references(:business_categories)
  end
```

In the view, we can then loop through @companies and display each one.

I was going to talk with @thesuss about this before doing anything with my code. (I thought she might want to do it after we went over it.)  But since you asked, Pat, I thought I'd go ahead and respond with it here. :-)

_(BTW -- the line in the above `show` method is the kind of thing I should be using in `Company#categories`.  ref: #82 )_